### PR TITLE
Soft landing page on projects or last viewed project

### PR DESCRIPTION
**Pull request by andreas-unleash**
Signed-off-by: andreas-unleash <andreas@getunleash.ai>

This PR introduces a soft landing page to the last viewed project or to the project list (if there is more than 1 project)

Changes: 
- Replaced clearing of `storage` with clearing `cache` in logout.ts :: REVERTED
- Root redirects to `projects` instead of `features`

## About the changes

Closes #

### CODE REVIEW

**Review by FredrikOseberg — COMMENTED** [review]
I think it looks good. I've added two questions that might impact how we do this.

File: frontend/src/hooks/useLastViewedProject.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { useEffect, useState } from 'react';
+
+export const useLastViewedProject = () => {
+    const [lastViewed, setLastViewed] = useState(() => {
+        return localStorage.getItem('lastViewedProject');
+    });
```
[the remark this answers is not in the document]
Comment (andreas-unleash) [reply frontend/src/hooks/useLastViewedProject.ts:6]: After a discussion with @sjaanus and @kwasniew , we decided to go with this approach until we find the need to store user settings in the BE.  Once that is implemented, we can swap out the call.

File: src/lib/routes/logout.ts [hunk 2]
```diff
@@ -59,7 +59,7 @@ class LogoutController extends Controller {
         res.clearCookie(this.cookieName);
 
         if (this.clearSiteDataOnLogout) {
-            res.set('Clear-Site-Data', '"cookies", "storage"');
+            res.set('Clear-Site-Data', '"cookies", "cache"');
```
[the remark this answers is not in the document]
Comment (andreas-unleash) [reply src/lib/routes/logout.ts:62]: The only things we store in localstorage are sorting for tables and this param
Screenshot 2022-11-23 at 09 35 42

**Review by FredrikOseberg — APPROVED** [review]
LGTM

**Pull request merged**